protocol/x/clob/keeper/process_operations_liquidations.go

```diff
@@ -11,6 +11,56 @@ import (
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// ValidateLiquidationOrderAgainstProposedMatch performs stateless validation of a liquidation order
+// against a proposed match.
+// An error is returned when
+//   - The CLOB pair IDs of the order and proposed match do not match.
+//   - The perpetual IDs of the order and proposed match do not match.
+//   - The size of the order is less than the size of the proposed match.
+//   - The side of the order and proposed match do not match.
+func (k Keeper) ValidateLiquidationOrderAgainstProposedMatch(
+	ctx sdk.Context,
+	order *types.LiquidationOrder,
```

Comment:
call validate on all the subaccount ids in the liquidation order and the liquidation match?

Reply:
This should be done in validate basic, right? 

we do call this for all subaccount ids except for match.Liquidated. added